packages/api/lib/xunit-parser.js

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const xmljs = require('xml-js');
+const TestCaseRun = require('../lib/testrun');
+
+class Parser {
+  parse (xmlString) {
+    const obj = xmljs.xml2js(xmlString, { compact: true });
+    const tests = [];
+    // Python doesn't always have a top-level testsuites element.
+    let testsuites = obj.testsuite;
+    if (testsuites === undefined) {
+      testsuites = obj.testsuites.testsuite;
+    }
+    if (testsuites === undefined) {
+      return tests;
+    }
+    // If there is only one test suite, put it into an array to make it iterable.
+    if (!Array.isArray(testsuites)) {
+      testsuites = [testsuites];
+    }
+    for (const suite of testsuites) {
+    // Ruby doesn't always have _attributes.
+      let testsuiteName = suite._attributes ? suite._attributes.name : undefined;
+
+      // Get rid of github.com/orgName/repoName/
```

Comment:
You could probably use regex here for this: 

```javascript
function trim(url) {
  return url.replace(/(.)*github.com\/[a-zA-Z\-]*\/[a-zA-Z\-]\/*/g, '');
}

console.log(trim("https://github.com/GoogleCloudPlatform/flaky-service/testName")); // prints "testName"
console.log(trim("github.com/GoogleCloudPlatform/flaky-service/testName")); // prints "testName"
```

Reply:
Ah, thank you! Just fixed that.